protos/dcs/srs/v0/srs.proto

```diff
@@ -86,3 +89,18 @@ message TransmitResponse {
   // The duration in milliseconds it roughly takes to speak the transmission.
   uint32 duration_ms = 1;
 }
+
+message GetClientsRequest {
+}
+
+message GetClientsResponse {
+  message Client {
+    // The unit that is connected to SRS.
+    dcs.common.v0.Unit unit = 1;
+
+    // The radio frequencies in Hz the unit is connected to.
+    repeated uint64 frequencies = 2;
```

Comment:
Could we add the unit? Is it in Hz? MHz?

Reply:
huh, it is in there in this view... was I looking at an older diff? Either way, looks good to me.